thavalon-server/src/game/state/proposing.rs

```diff
@@ -6,9 +6,43 @@ use super::prelude::*;
 pub struct Proposing {
     /// The player whose turn it is to propose
     proposer: String,
+    selected_players: HashSet<String>,
 }
 
 impl GameState<Proposing> {
+    pub fn handle_player_selected(mut self, player: &str, added_player: String) -> ActionResult {
```

Comment:
I've been really inconsistent in this, but should this pub fn have documentation? If you haven't added it anywhere in this module, probably fine to not.

Reply:
I've been pretty inconsistent too, but I can add some